core/src/main/java/com/novoda/rxpresso/mock/SimpleEvents.java

```diff
@@ -0,0 +1,36 @@
+package com.novoda.rxpresso.mock;
+
+import rx.Observable;
+
+public final class SimpleEvents {
```

Comment:
SingleEvent?

we've used SimpleX in the past to name base classes for when we don't want to implement all the methods in an interface. SingleEvent might also negate the need to have the comment on `onNext()`